microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-105.md

```diff
@@ -0,0 +1,180 @@
+
+You can now change the orientation of the Quick View panel as well. To change Quick View to a vertical split, select the location toggle (**Dock Quick View to the right**). Selecting this toggle a second time will return Quick View to a horizontal split.
+
+![2](./devtools-105-images/quickview-console.png) 
+
+<!-- 
+Screenshot 2: Ensure Focus Mode is enabled. With Elements open in main panel and Console in Quick View, press the location toggle at the top right of Quick View to change its orientation to vertical. -->
+
+To minimize Quick View, use the **Collapse** toggle or press `Escape`:
+
+![3](./devtools-105-images/focus-mode-improved-location-controls3.png)
+<!-- Screenshot 3: Continuation of previous screenshot. Minimize Quick View in vertical orientation with with Esc keyboard shortcut. See below. -->
+
+See also:
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+<!-- ====================================================================== -->
+
+## Fix: Search in the Sources and Network tools now works as expected in Focus Mode
+
+<!-- Rank 4 -->
+<!-- Title: Fix: Search in the Sources and Network tools now works as expected in Focus Mode  -->
+<!-- Subtitle: Try Focus Mode, a new, more simplified, and streamlined UI for DevTools!-->
+
+[Focus Mode experiment](https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/experimental-features/focus-mode#enable-focus-mode) in previous versions of Microsoft Edge didn't properly showcase the Search feature in the Network tool nor the *Search in all files* feature in the Sources tool. In Microsoft Edge version 105, these issues have been fixed.
+
+You can now search in the Network tool via the Search icon.  
+
+![1](./devtools-105-images/networking-search.png)
+
+You can leverage the *Search in all files* feature in the Sources tool by opening the context menu on files in the Page section.  This will display the Search tool in the Quick View.
+
+![2](./devtools-105-images/networking-search2.png)
+
+<!-- Open Edge Beta, Dev, or Canary (navigate to edge://version and ensure you're on 105+)
+1. Navigate to:  Simplify DevTools using Focus Mode - Microsoft Edge Development | Microsoft Docs
+2. Open DevTools. Go to Settings > Experiments. Turn on Focus Mode. Reload DevTools
+3. Open the Network tool. Refresh the page so you can see some network requests. Now select the Search (magnifying glass icon)
+4. Take a screenshot
+5. Open the Sources tool
+6. Right-click docs.microsoft.com
+7. Take a screenshot of the "Search in all files" context menu entry
+8. Draw red highlight boxes around the Search icon and Search pane in the Network tool. Draw a red highlight box around the "Search in all files" entry in the context menu in the Sources tool. -->
+
+
+See also:
+* [Inspect network activity - Microsoft Edge Development](https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/network/#search-network-headers-and-responses)
```

Comment:
If this link is supposed to go to section anchor, replace link by:
* [Search network headers and responses](https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/network/#search-network-headers-and-responses) in _Inspect network activity_.

If this link is supposed to go to whole article, remove anchor from link:
* [Inspect network activity](https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/network/)